mods/parsoid.js

```diff
@@ -264,7 +269,7 @@ function sameHtml(a, b) {
     return normalizeHtml(a) === normalizeHtml(b);
 }
 
-PSP.generateAndSave = function(restbase, req, format, currentContentRes) {
+PSP.generateAndSave = function(restbase, req, format, currentContentRes, currentContentTid) {
```

Comment:
Why is `currentContentTid` needed? We can get it from `req.params.tid`, can't we?